assembl/static2/js/app/pages/discussionAdmin.jsx

```diff
@@ -270,6 +271,24 @@ const mapStateToProps: MapStateToProps<ReduxState, *, *> = ({
 }) => {
   const { sectionsById, sectionsHaveChanged, sectionsInOrder } = sections;
   const { profileOptionsHasChanged, textFieldsById } = profileOptions;
+  const textFields = textFieldsById
+    .map(textField => textField)
+    .valueSeq()
+    .toJS();
+  textFields.forEach((field) => {
+    const newField = field;
+    if (field.options) {
+      // convert options to array, and remove _hasChanged on option that is set by moveItemUp/Down.
```

Comment:
Why? Apollo doesn't like a key that shouldn't be there?

Reply:
exactly, _hasChanged is not part of SelectFieldOptionInput